README.md

```diff
@@ -80,15 +80,15 @@ https://github.com/jcubic/lips/blob/master/lib/js/bookmark.js
 Create any link in your bookmarks, edit it and copy paste the content of that file.
 Affter you click on the link it will create the REPL at the bottom of the page.
 (NOTE: It may not work on every page because of content security policy;
-e.g. google.com or gihub.com)
+e.g. google.com or github.com)
 
-If you have trouble with creating the bookmarklet you can open
-[LISP Scheme home page](https://lips.js.org/#bookmark) where you can
+If you have trouble with creating the bookmark you can open
```

Comment:
[bookmarklet](https://en.wikipedia.org/wiki/Bookmarklet) is the name of the bookmark that is also JavaScript code that you can execute.